solidity/contracts/bridge/Bridge.sol

```diff
@@ -2133,13 +1657,30 @@ contract Bridge is Ownable, EcdsaWalletOwner {
     ///         For example, if the treasury fee needs to be 2% of each deposit,
     ///         the `depositTreasuryFeeDivisor` should be set to `50`
     ///         because `1/50 = 0.02 = 2%`.
+    /// @return depositTxMaxFee Maximum amount of BTC transaction fee that can
+    ///         be incurred by each swept deposit being part of the given sweep
+    ///         transaction. If the maximum BTC transaction fee is exceeded,
+    ///         such transaction is considered a fraud.
+    /// @return treasury Address where the deposit treasury fees will be
+    ///         sent to. Treasury takes part in the operators rewarding process.
+    /// @return txProofDifficultyFactor The number of confirmations on the
+    ///         Bitcoin chain required to successfully evaluate an SPV proof.
     function depositParameters()
         external
         view
-        returns (uint64 depositDustThreshold, uint64 depositTreasuryFeeDivisor)
+        returns (
+            uint64 depositDustThreshold,
+            uint64 depositTreasuryFeeDivisor,
+            uint64 depositTxMaxFee,
+            address treasury,
```

Comment:
The `treasury` is also used for redemptions. Shouldn't it live under `getContracts`? I'm aware the treasury will not be necessarily a contract but it seems to be something like "an address of external dependency" not just a "deposit parameter".

Reply:
Let's talk about it in #206. I extracted it there into a separate function but I can also move it to `getContracts`. The thing is that treasury is not necessarily a contract.